File: 프로그래머스/연습문제/Lv1/바탕화면 정리/바탕화면 정리.py
def solution(wallpaper):
    start_i,start_j = 52,52
    end_i,end_j = 0,0

    for i in range(len(wallpaper)):
        for j in range(len(wallpaper[0])):
            if wallpaper[i][j] == '#':
                start_i = min(start_i, i)
                start_j = min(start_j, j)
                end_i = max(end_i, i)
                end_j = max(end_j, j)

    return [start_i, start_j, end_i+1, end_j+1]

File: 프로그래머스/연습문제/Lv1/바탕화면 정리/바탕화면 정리.cpp
#include <string>
#include <vector>

using namespace std;

vector<int> solution(vector<string> wallpaper) {
    int start_i=52, start_j=52, end_i=0, end_j=0;
    for(int i=0; i<wallpaper.size(); i++){
        for(int j=0; j<wallpaper[0].size(); j++){
            if(wallpaper[i][j] == '#'){
                if(i < start_i) start_i = i;
                if(j < start_j) start_j = j;
                if(i > end_i) end_i = i;
                if(j > end_j) end_j = j;
            }
        }
    }
    
    return {start_i, start_j, end_i+1, end_j+1};
}
